pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
+
+`brightnessctl --list`
+
+- Get the currently set brightness of the display backlight:
+
+`brightnessctl get`
+
+- Set the brightness of the display backlight to certain percent of the maximum:
```

Comment:
> 
Perhaps this will read better?
```suggestion
- Set the brightness of the display backlight to a percenage of the maximum supported value:
```

Reply:
As I said [here](https://github.com/tldr-pages/tldr/pull/6516#discussion_r708198885), I think we can shorten that down to
```suggestion
- Set the brightness of the display backlight:
```

Because brighness is almost always given in percent, so it's the behaviour you'd expect anyways.